NEWS.md

```diff
@@ -3,10 +3,11 @@ htmlwidgets 1.5.4.9000
 
 ### Breaking change
 
-* `shinyWidgetOutput()`'s `reportSize` argument now defaults to `TRUE`. This way, calling `shiny::getCurrentOutputInfo()` inside a `shinyRenderWidget()` context will report the current height and width of the widget. 
+* `shinyWidgetOutput()`'s `reportSize` argument now defaults to `TRUE`. This way, calling `shiny::getCurrentOutputInfo()` inside a `shinyRenderWidget()` context will report the current height and width of the widget.
 
 ### Improvements
 
+* `shinyWidgetOutput()` and `sizingPolicy()` both gain a new `fill` parameter. When `TRUE`, the widget is allowed to grow/shrink to fit it's parent container so long as that parent has a defined height and has been marked with `htmltools::asFillContainer()` (https://github.com/rstudio/htmltools/pull/343). The primary motivation for this is to allow widgets to grow and shrink (by default) when used [inside `bslib::card_body_fill()`](https://rstudio.github.io/bslib/articles/cards.html#responsive-sizing) (#442)
```

Comment:
Some aspect of this is potentially a breaking change, right? If you maintain a widget that has `inline=FALSE` but is not responsive to increased height, then you need to modify your call to `shinyWidgetOutput()` to set `fill = FALSE`.